build/lint.cloudbuild.yaml

```diff
@@ -21,4 +21,7 @@ tags:
 - 'lint'
 substitutions:
   _DOCKER_IMAGE_DEVELOPER_TOOLS: 'cft/developer-tools'
-  _DOCKER_TAG_VERSION_DEVELOPER_TOOLS: '1.7'
+  _DOCKER_TAG_VERSION_DEVELOPER_TOOLS: '1'
+options:
+  env:
+  - DISABLE_TFLINT=1
```

Comment:
Todo in https://github.com/GoogleCloudPlatform/terraform-google-cloud-run/issues/100 as it will touch other modules.